File: CPP/Address.cpp
//
// Created by Nutzer on 22.08.2025.
//

#include "Address.h"

const std::string& Address::getFormattedAddress() const {
    return formattedAddress;
}

void Address::addLine(const std::string& line) {
    formattedAddress += line;
}

File: Python/Address.py
class Address:
    def __init__(self, address: str = ""):
        self._formattedAddress = address

    @property
    def formatted_address(self) -> str:
        return self._formattedAddress

    def add_line(self, line: str) -> None:
        self._formattedAddress += line
